### PR TITLE
add a dict for status + bug fix

### PR DESCRIPTION
**Pull request by j4mesjung**

### CODE REVIEW

File: src/app/model/Mission.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import { get, filter } from "lodash";
 
+export const missionStatusDict = {
```
Comment (mat10tng) [inline src/app/model/Mission.js:3]: Maybe throw this in mission model while you at it

Comment (j4mesjung) [reply src/app/model/Mission.js:3]: hi @mat10tng, not sure what you mean, can you please provide example?

**Comment by thirdeyeclub** [comment]
You seem to have a merge conflict

**Comment by j4mesjung** [comment]
accidentally rebased, going to close and open in new branch.

**Comment by mat10tng** [comment]
Hi just a reminder that you should be using npm ci instead of npm install

Also, about the model
image
You could put a function inthere to select the correct term

**Pull request closed without merging**